homeassistant/components/nmbs/sensor.py

```diff
@@ -92,14 +92,19 @@ def __init__(self, api_client, live_station):
         """Initialize the sensor for getting liveboard data."""
         self._station = live_station
         self._api_client = api_client
-
+        self._unique_id = f"nmbs_live_{self._station}"
         self._attrs = {}
         self._state = None
 
     @property
     def name(self):
         """Return the sensor default name."""
-        return "NMBS Live"
+        return f"NMBS Live {self._station}"
```

Comment:
If we don't change the name now, we can avoid a breaking change.

Reply:
Fixed with https://github.com/home-assistant/home-assistant/pull/29662/commits/969cf2ff767a687fb6e07c2f227b0cf02cef44d0